.vscode/settings.json

```diff
@@ -25,9 +25,9 @@
   "[python]": {
     "editor.defaultFormatter": "ms-python.black-formatter",
     "editor.formatOnSave": true,
-    "editor.rulers": [150]
+    "editor.rulers": [79]
   },
-  "black-formatter.args": ["--line-length=150"],
+  "black-formatter.args": ["--line-length=79"],
```

Comment:
79 feels a bit short sometimes, the pep doc linked said that 99 was okay

Reply:
https://lastmileai.dev/workbooks/clrqqjxg90534qp8jin49nxx9